src/styles/button.css

```diff
@@ -1 +1,71 @@
 /* button */
+
+.ui.button {
+  background-color: var(--primary-pms-362-900);
+  color: var(--shade-white);
+  border-radius: var(--border-radius-3);
+  padding: 0.625rem;
+  line-height: 1.5em;
```

Comment:
Shouldn't the line height change with the font size? eg. when fon-size is 1.8, line-height is 1.3

Reply:
Let's add the `line-height` values explicitly, just in case.

Also, one last change - for some reason Semantic adds margins to buttons by default (very strange!). Let's remove them, along with the `min-height`.

```
min-height: 0;
margin: 0;
```